tests/CMakeLists.txt

```diff
@@ -147,3 +172,45 @@ if (NOT (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_CURRENT_BINARY_DIR) OR MSVC)
 
   add_custom_target(copy-tests ALL DEPENDS ${TEST_FILES_OUT})
 endif()
+
+if(DEFINED SKBUILD)
+  if(NOT CMAKE_CROSSCOMPILING)
+    install(
+      FILES
+        "${CMAKE_CURRENT_BINARY_DIR}/test_functions_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_stl_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_typing_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_enum_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_ndarray_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_make_iterator_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/test_classes_ext.pyi"
+        "${CMAKE_CURRENT_BINARY_DIR}/py_stub_test.pyi"
+      DESTINATION
+        .
+    )
+  endif()
+  install(
+    TARGETS
+      inter_module
```

Comment:
There is a bit too much repetition of test names in this project. How about defining them as a list and then creating the build and install targets by pasting the list or iterating over elements.

Reply:
I love `list(TRANSFORM` (CMake 3.12+) :)